src/screen/TreasuryScreen.tsx

```diff
@@ -13,11 +13,13 @@ import Identicon from '@polkadot/reactnative-identicon';
 import {SafeAreaView} from 'react-native-safe-area-context';
 import {getAccountsIdentityInfo} from 'service/api/account';
 import {EmptyView} from 'presentational/EmptyView';
+import {useFormatBalance} from '../hook/useFormatBalance';
 
 export function TreasuryScreen({navigation}: {navigation: NavigationProp<DashboardStackParamList>}) {
+  const theme = useTheme();
   const {api} = useContext(ChainApiContext);
 
-  const {loading, value: treasuryData} = useAsyncRetry(async () => {
+  const {loading, value, retry} = useAsyncRetry(async () => {
```

Comment:
Maybe we should create an issue to replace the usages of this hook in favor of useQuery from react-query for api related hooks

Reply:
#154 ✅